src/focus-trap-react.js

```diff
@@ -183,7 +199,11 @@ FocusTrap.propTypes = {
   paused: PropTypes.bool,
   focusTrapOptions: PropTypes.shape({
```

Comment:
Can these prop type options be moved into the core focus-trap library instead of having to replicate them in all the spin-off focus-trap projects?

It will make maintaining them much easier.

Reply:
Ok, yeah PropTypes is more of a React thing. 

It just means that it is not as simple as just installing a new focus-trap version when the settings change.

By spin-off projects I was thinking there would also be Vue, Angular, Svelt etc. versions of the project.